src/QsCompiler/BondSchemas/BondSchemaTranslator.cs

```diff
@@ -374,7 +374,7 @@ qsResult.Tag switch
                 Kind = qsSpecialization.Kind.ToBondSchema(),
                 Parent = qsSpecialization.Parent.ToBondSchema(),
                 Attributes = qsSpecialization.Attributes.Select(a => a.ToBondSchema()).ToList(),
-                SourceFile = qsSpecialization.SourceFile,
+                SourceFile = qsSpecialization.Source.CodeFile,
```

Comment:
We are losing a bit of data from the `Source` object by just including the value of `CodeFile`. This is consistent with what we had before but we would not have the additional information from loaded references. Is this by design?

Reply:
Yes, because `AssemblyFile` is always set to the DLL file path after deserializing a reference, I can't see any situation in which a serialized `AssemblyFile` property would be used.